WordPress/gutenberg
# Table of contents block accessibility improvements

## What?


This PR fixes some accessibility issues in the table of contents block, front-end and back-end.

## Why?


Accessibility is important.

## How?


- Remove inert.
- Disable the ability to click links in the back-end block.
- Add `aria-label` to front-end `<nav>` tag.

## Testing Instructions





1. Insert a Table of Contents block.
2. Verify the block is keyboard accessible with arrow keys.
3. Verify clicking the links is disabled.
4. Verify that screen readers get a disabled announcement through `aria-disabled="true"` attribute.
5. Check the front end for the `aria-label="Table of Contents"` attribute.

### Testing Instructions for Keyboard


Verify in the back-end that Up and Down Arrow keys allow you to navigate the block content.

## Screenshots or screencast
